build/helper/codegen_helper.py

```diff
@@ -146,13 +146,14 @@ def get_ctype_variable_declaration_snippet(parameter, parameters, config):
        11. Output buffer with mechanism ivi-dance:                      None
        12. Output buffer with mechanism passed-in:                      (visatype.ViInt32 * buffer_size)()
        13. Output scalar or enum:                                       visatype.ViInt32()
+       14. Input buffer (custom type):                                  (custom_struct * len(list))(*[custom_struct(l) for l in list])
```

Comment:
As I wrote this, I made sure to keep this list and the code below in the same order... so 14 should be 5... of course this means everything else needs to be renumbered. The way I would do it (it took many passes) is to add the new thing as 4.5, commit and make sure everything works, then renumber in a different commit.

Reply:
Renumbered